pyInterface/userInterface/calcAmplitudes.py

```diff
@@ -72,6 +73,10 @@
 		pyRootPwa.utils.printErr("Invalid events type given ('" + args.eventsType + "'). Aborting...")
 		sys.exit(1)
 
+	if args.debugFlag:
+		pyRootPwa.core.isobarDecayTopology.debugIsobarAmplitude = True
+		pyRootPwa.core.isobarDecayTopology.debugIsobarDecayTopology = True
+
```

Comment:
I do not think it is worth to change this file. There are a myriad of debug flags and the ones which need to be enabled depends strongly on the problem one is looking at.